aws/rust-runtime/aws-config/src/profile/credentials.rs

```diff
@@ -364,6 +364,16 @@ impl Builder {
                     conf.env(),
                 ))
             });
+
+        named_providers
+            .entry("Ec2InstanceMetadata".into())
+            .or_insert_with(|| {
```

Comment:
Does this need to be behind a feature flag?

Reply:
no, but good catch, I missed adding `imds` to the profile feature